src/modules/45-dashboards/types/DashboardTypes.tsx

```diff
@@ -35,3 +35,10 @@ export enum DashboardLayoutViews {
   LIST,
   GRID
 }
+
+export interface IDashboardFormData {
```

Comment:
We don't typically use Hungarian notation in NGUI
```suggestion
export interface DashboardFormData {
```